modules/phase_field/src/actions/PolycrystalColoringICAction.C

```diff
@@ -19,20 +19,29 @@ InputParameters
 PolycrystalColoringICAction::validParams()
 {
   InputParameters params = Action::validParams();
-  params.addClassDescription("Random Voronoi tessellation polycrystal action");
+  params.addClassDescription(
+      "Action to create ICs for polycrystal variables defined using a Voronoi tesselation");
   params.addRequiredParam<unsigned int>("op_num", "number of order parameters to create");
   params.addRequiredParam<std::string>("var_name_base", "specifies the base name of the variables");
   params.addRequiredParam<UserObjectName>("polycrystal_ic_uo", "Optional: TODO");
   params.addParam<std::vector<SubdomainName>>("block",
                                               "Block restriction for the initial condition");
+  params.addParam<bool>("nonlinear_preconditioning",
+                        false,
+                        "Whether to use nonlinear preconditioning or the standard model");
+  params.addParam<Real>("bound_value",
+                        5.0,
+                        "ONLY FOR NONLINEAR PRECONDITIONING: Bound value used to keep variable "
```

Comment:
Rather than the all cap comment how about putting both parameters into a `Nonlinear Preconditioning` parameter group?

Reply:
@dschwen I can't remember the syntax for parameter groups. Can you point me at an example?

Reply:
params.addParamNamesToGroup("shift scale", "group name");

Reply:
I have addressed this comment